src/categorical_algebra/FinCats.jl

```diff
@@ -331,6 +332,47 @@ FinFunctor(ob_map, hom_map, dom::Presentation, codom::Presentation) =
 Categories.show_type_constructor(io::IO, ::Type{<:FinFunctor}) =
   print(io, "FinFunctor")
 
+function is_initial(F::FinFunctor)::Bool
```

Comment:
Function should have a docstring. Would also be helpful to link to the nlab page for "final functor" since this concept is a bit obscure.